Cogs/website.py

```diff
@@ -264,6 +264,84 @@ async def addplayer_error(self, ctx: discord.context.ApplicationContext, error):
             await ctx.respond(embed = discord.Embed(title = "Unknown error. Please contact developers to check logs", color = discord.Color.red()))
             print("Add Player error: ", error)
             raise error
+###########################################################################
+    @discord.slash_command(description = "Deletes a Player", debug_guilds=[887366492730036276])
+    @commands.has_any_role('Board Member', 'Game Officer', 'Bot Technician', 'Mod', 'Faculty Advisor')
+    async def deleteplayer(
+        self,
+        ctx: discord.context.ApplicationContext,
+        playerName: discord.Option(str, "Select the Player to delete", name="player", autocomplete=create_player_autocomplete(player_cache))
+    ):
+        await ctx.defer()
+
+        player = await player_cache.find(playerName)
+        if player == None:
+            await ctx.respond(f"Could not find a player named {playerName} (Try using the autocomplete!)")
```

Comment:
Didn't catch this until my 2nd time through, but would probably be useful to change all the .respond()'s to use embeds, since everything else we have is also an embed.